.storybook/main.js

```diff
@@ -2,12 +2,21 @@ const path = require('path')
 
 module.exports = {
   stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
-  addons: [
-    '@storybook/addon-links',
-    '@storybook/addon-essentials',
-    'storybook-css-modules-preset',
-  ],
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
   webpackFinal: config => {
+    config.module.rules.push({
+      test: /\.css&/,
+      use: [
+        {
+          loader: 'postcss-loader',
+          options: {
+            ident: 'postcss',
+            plugins: [require('tailwindcss'), require('autoprefixer')],
+          },
+        },
+      ],
+      include: path.resolve(__dirname, '../'),
+    })
```

Comment:
Could you please test storybook one more time without these changes?
For me storybook works well with both tailwind and css modules

Reply:
It doesn't work on windows without this. What OS did you work on?

Reply:
linux

Reply:
And it is necessary to import the general css file